osm/src/main/groovy/org/orbisgis/osm/Loader.groovy

```diff
@@ -228,61 +213,34 @@ IProcess load() {
         title "Load an OSM file to the current database"
         inputs datasource: JdbcDataSource, osmTablesPrefix: String, osmFilePath: String
         outputs datasource: JdbcDataSource
-        run { datasource, osmTablesPrefix, osmFilePath ->
-            if (datasource != null) {
-                info "Load the OSM file in the database"
-                File osmFile = new File(osmFilePath)
-                if (osmFile.exists()) {
-                    datasource.load(osmFile, osmTablesPrefix, true)
-                    info "The input OSM file has been loaded in the database"
-                } else {
-                    error "The input OSM file does not exist"
-                    return null
-                }
-            } else {
-                error "Please set a valid database connection"
-                return null
+        run { JdbcDataSource datasource, osmTablesPrefix, osmFilePath ->
+            if(!datasource) {
+                error "Please set a valid database connection."
+                return
             }
-            [datasource: datasource]
```

Comment:
removed ?

Reply:
No, it's git which doesn't show well the code modification. Checkout a little further, there is the rest of the function.